src/ui/wxWidgets/PasswordSafeSearch.cpp

```diff
@@ -64,33 +66,34 @@ enum {
   ID_FIND_CREATE_REPORT,
   ID_FIND_CLEAR,
   ID_FIND_STATUS_AREA
-};
+}; // see also PasswordSafeSearch::CalculateToolsWidth() in case of updates
 
 PasswordSafeSearch::PasswordSafeSearch(PasswordSafeFrame* parent) : m_toolbar(0),
                                                                     m_parentFrame(parent),
                                                                     m_criteria(new SelectionCriteria)
 {
 }
 
-PasswordSafeSearch::~PasswordSafeSearch(void)
+PasswordSafeSearch::~PasswordSafeSearch()
 {
   delete m_toolbar;
   m_toolbar = 0;
+
   delete m_criteria;
   m_criteria = 0;
 }
 
-void PasswordSafeSearch::OnSearchTextChanged(wxCommandEvent& evt)
+void PasswordSafeSearch::OnSearchTextChanged(wxCommandEvent& event)
 {
-  wxSearchCtrl *srchCtrl = wxDynamicCast(evt.GetEventObject(), wxSearchCtrl);
+  wxSearchCtrl *srchCtrl = wxDynamicCast(event.GetEventObject(), wxSearchCtrl);
   srchCtrl->SetModified(true);
```

Comment:
Since a dynamic cast can fail, worth asserting here that srchCtrl != nullptr, no?

Reply:
Fixed by 21a5714f5880b395422349110646fbd0ff41258c